README.md

```diff
@@ -289,6 +289,18 @@ dependencies, without committing, tagging, pushing or publishing.
 
 > Only update versions and dependencies; don't actually publish (this only affects publish).
 
+#### --registry [registry]
+
+```sh
+$ lerna publish --registry https://my-private-registry
```

Comment:
I might even suggest that _all_ npm operations provide the registry flag -- I think that's what I would expect if I provided a registry key in lerna.json.